oam/install_scripts/post-install

```diff
@@ -234,31 +254,49 @@ if [ $user = "root" ]; then
 fi
 
 if [ $user = "root" ]; then
-	$installdir/bin/syslogSetup.sh install > /tmp/syslog_install.log 2>&1
-	rm -f /etc/default/columnstore
+	$installdir/bin/syslogSetup.sh install > $tmpDir/syslog_install.log 2>&1
+
+	#check if MariaDB Columnstore system logging was setup
+	cat $tmpDir/syslog_install.log | grep 'No System Logging' >/dev/null 2>&1
+	if [ $? -eq 0 ]; then
+	    cat $tmpDir/syslog_install.log
+	fi
 else
-        sed -i -e s@/usr/local/mariadb/columnstore@$installdir@g $installdir/bin/columnstore.def
-        sed -i -e s@/usr/local/mariadb/columnstore@$installdir@g $installdir/bin/columnstoreLogRotate
+	sed -i -e s@/usr/local/mariadb/columnstore@$installdir@g $installdir/bin/columnstoreLogRotate
 
-        $SUDO cp $installdir/bin/columnstore.def /etc/default/columnstore
+	sed -i -e s@prefix=/home/guest@prefix=$prefix@g $installdir/bin/*
 
-        sed -i -e s@prefix=/home/guest@prefix=$prefix@g $installdir/bin/*
+	chown $user:$user $installdir/etc/Columnstore.xml
+	
+cat <<EOD
 
-	$SUDO chmod 777 /tmp
-	$installdir/bin/syslogSetup.sh --installdir=$installdir install > /tmp/syslog_install.log 2>&1
-	$SUDO chown $user:$user $installdir/etc/Columnstore.xml
-	$SUDO mkdir /var/lock/subsys > /dev/null 2>&1
-	$SUDO chmod 777 /var/lock/subsys > /dev/null 2>&1
-	$SUDO rm -f /var/lock/subsys/mysql-Columnstore
-	$SUDO chmod 666 /etc/fstab
+NOTE: For non-root install, you will need to run the following commands as root user to
+      setup the MariaDB ColumnStore System Logging
+
+export COLUMNSTORE_INSTALL_DIR=$COLUMNSTORE_INSTALL_DIR
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH
```

Comment:
Non-critical issue.  Whenever I've run post-configure, I get LD_LIBRARY_PATH equal to 3 or 4 copies of what it should be.